theme/material/vaadin-grid-styles.html

```diff
@@ -112,6 +112,10 @@
         right: -1px;
       }
 
+      [part~="cell"] [part="resize-handle"] {
+        position: initial;
```

Comment:
Changing the resize handles' position to `static` will actually affect the header cell sizing which might misalign the content. Would removing the `right: -1px` above instead be a cleaner fix?